src/Auxiliary/Aux_ComputeProfile.cpp

```diff
@@ -123,6 +127,13 @@ void Aux_ComputeProfile( Profile_t *Prof[], const double Center[], const double
    }
 
 
+// check whether _POTE is in TVarBitIdx
+   bool InclPot = false;
+
+   for (int p=0; p<NProf; p++)
+      if ( TVarBitIdx[p] & _POTE )   InclPot = true;
```

Comment:
Enclosed by `#ifdef GRAVITY ... #endif`

Reply:
In addition, it's a bit confusing why checking only `_POTE` but not other fields like fluid and magnetic field.

Reply:
The InclPot is used to avoid determining the interpolation parameters of potential at the beginning in which the potential is not initialized (and the times are still -1). This might be removed in a future version where GREP is treated as external potential/acceleration.

In my opinion, I prefer all the interpolation parameters should be initialized even not required (like _POTE) because this step is relatively cheap (call `SetTempIntPara` once for each field per level). What do you think?

Reply:
I agree with you. I would suggest adding comments about why we need `InclPot` (e.g., something like "potential data may have not been computed during initialization").

Reply:
Updated. I also add `REVISE` comment to remind me of further modifications in the future version.